Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Controllers/UsersController.cs

```diff
@@ -39,22 +36,37 @@ public async Task<IActionResult> GetEntry([FromRoute] int id)
             await _db.SaveChangesAsync();
             return Ok(entry);
         }
-        // POST api/values
+        [HttpPost("login")]
+        public ActionResult<Users> Login([FromBody] Users Body)
+        {
```

Comment:
Since you've asked me in pvt to add the comments on PR so there is a whitespace here

Reply:
whoops didnt see that thanks